com.unity.netcode.gameobjects/Runtime/NetworkVariable/NetworkVariableBase.cs

```diff
@@ -13,14 +13,11 @@ public void Initialize(NetworkBehaviour networkBehaviour)
         {
             m_NetworkBehaviour = networkBehaviour;
         }
-        /// <summary>
-        /// The name of the channel to be used for syncing
-        /// </summary>
-        public const NetworkChannel NetworkVariableChannel = NetworkChannel.NetworkVariable;
 
-        protected NetworkVariableBase(NetworkVariableReadPermission readPermIn = NetworkVariableReadPermission.Everyone)
+        protected NetworkVariableBase(NetworkVariableReadPermission readPermIn = NetworkVariableReadPermission.Everyone, NetworkDelivery networkDelivery = NetworkDelivery.ReliableSequenced)
```

Comment:
@mattwalsh-unity 
making `NetworkDelivery` of a `NetworkVariable` via the constructor was requested by @SamuelBellomo and currently we have no technical reasons to not allow for this. he'd like to use this ctor to configure position updates to be unreliable but other state data suich as health reliable.

Reply:
Yeah, like we chatted about, we want to avoid suggesting to users that we're really going to support this beyond this release.  Netvars all go through snapshotting, and snapshotting will have one and only one delivery for all netvars, which effectively is "eventually reliable".